### PR TITLE
DM-42054: Remove limit of 1 for getCommandsDuringEvent

### PR DESCRIPTION
**Pull request by mfisherlevine**

### CODE REVIEW

File: python/lsst/summit/utils/efdUtils.py [hunk 1]
```diff
@@ -661,3 +662,76 @@ def getTopics(client, toFind, caseSensitive=False):
...
+    Raises
+    ------
+    ValueError
+        Raise if there is already a command at a timestamp in the dictionary,
+        i.e. there is a collision.
+    """
+    if timeFormat not in ['pandas', 'astropy', 'python']:
+        raise ValueError(f"format must be one of 'pandas', 'astropy' or 'python', not {timeFormat=}")
+
+    commands = list(ensure_iterable(commands))
```
Comment (fritzm) [inline python/lsst/summit/utils/efdUtils.py:705]: Why "listify" here?  Can't the iterable be directly used in the `for` loop below?

File: python/lsst/summit/utils/tmaUtils.py [hunk 2]
```diff
@@ -510,39 +540,29 @@ def getCommandsDuringEvent(client, event, commands=('raDecTarget'), log=None, do
...
-    # for triggering commands before the event
-
-    # TODO: DM-40100 Change this to always return a list of times, and remove
-    # warning about finding multiple commands. Remember to update docs and
-    # plotting code.
-    if log is None and doLog:
-        log = logging.getLogger(__name__)
-
-    commands = ensure_iterable(commands)
+    commands = list(ensure_iterable(commands))
```
Comment (fritzm) [inline python/lsst/summit/utils/tmaUtils.py:548]: same question as above...

**Review by fritzm — APPROVED** [review]
LGTM at a coding level, one minor question.  You may still want Tiago's eyes for possible impact to the broader codebase?

File: python/lsst/summit/utils/efdUtils.py (hunk 1 above)
Comment (mfisherlevine) [reply python/lsst/summit/utils/efdUtils.py:705]: I hate that too, but `ensure_iterable` will just return an exhaustible generator if it got one of those, and I hate accidentally consuming things when looping over them to find they're now gone, so just tend to do that for safety when I know the list is small, mainly just out of habit. It's not necessary here, but it's also probably less harmful than not doing it, on balance.

**Comment by mfisherlevine** [comment]
Thanks. Should be fine though, this can't affect anything in Tiago-town (expect making it easier for him to use this code to debug things 🙂)

**Pull request merged**